StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Vertical Main Screen/RowButton.swift

```diff
@@ -98,11 +97,22 @@ class RowButton: UIView {
             labelsStackView.topAnchor.constraint(greaterThanOrEqualTo: topAnchor, constant: 4),
             labelsStackView.bottomAnchor.constraint(lessThanOrEqualTo: bottomAnchor, constant: -4),
         ])
-        addGestureRecognizer(UITapGestureRecognizer(target: self, action: #selector(handleTap)))
+
+        if let rightAccessoryView {
+            rightAccessoryView.translatesAutoresizingMaskIntoConstraints = false
+            addSubview(rightAccessoryView)
+            NSLayoutConstraint.activate([
+                rightAccessoryView.topAnchor.constraint(equalTo: topAnchor),
+                rightAccessoryView.bottomAnchor.constraint(equalTo: bottomAnchor),
+                rightAccessoryView.trailingAnchor.constraint(equalTo: trailingAnchor),
```

Comment:
This is missing a leading anchor constraint - w/o this can overlap the other content in the button.

Reply:
I think the labels are actually missing their trailing anchor, the labels shouldn't overlap with the right accessory view or spill over if it has long text. I updated is so the labels trailing anchor is either based on the containing view or the right accessory view.
Before:
![Simulator Screenshot - iPhone 15 Plus - 2024-05-20 at 12 04 17](https://github.com/stripe/stripe-ios/assets/88012362/e812b0ab-3fc6-42bc-9f44-b6a222eb7be6)